components/board/fake/board.go

```diff
@@ -21,6 +21,10 @@ import (
 	"go.viam.com/rdk/resource"
 )
 
+// To see fake analog values on a fake board, add an analog reader on pin 4.
+// Only pin 4 is used for fake values in order to maintain test functionality.
```

Comment:
I don't understand this comment. All values on a fake board are fake; what happens on the other pins? Surely they're not real values, right? or maybe only pin 4 can be used for analogs and pins 1-3 don't exist or give errors or something?

(I think it's that pin 4 will auto-increment the value read, while all other pins will reflect back the most recent value written.)

Reply:
correct all other pins will just output 0. I'll add that